modules/create-data-object/build/create-data-object/Dockerfile

```diff
@@ -1,4 +1,19 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal AS builder
-RUN microdnf install -y golang-1.16.* && microdnf clean all
+RUN microdnf install -y tar gzip && microdnf clean all
+ENV TASK_NAME=create-data-object \
+    GOFLAGS="-mod=vendor"
+WORKDIR /src/${TASK_NAME}
+RUN curl -L https://go.dev/dl/go1.18.3.linux-amd64.tar.gz | tar -C /usr/local -xzf -
+ENV PATH=$PATH:/usr/local/go/bin
+COPY . .
+RUN	CGO_ENABLED=0 GOOS=linux go build -o /${TASK_NAME} cmd/${TASK_NAME}/main.go
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+ENV TASK_NAME=create-data-object
+ENV ENTRY_CMD=/usr/local/bin/${TASK_NAME}
+
```

Comment:
can you please add here command `microdnf update -y && microdnf clean all`

Reply:
Is that really needed in the final image? I could not find it in other task Dockerfiles.

Reply:
This will update image, so we can prevent some vulnerabilities. I will add it to other images as well

Reply:
Shouldn't the `latest` image already be up to date?